src/osdc/Objecter.cc

```diff
@@ -4795,7 +4794,6 @@ Objecter::~Objecter()
   assert(check_latest_map_ops.empty());
   assert(check_latest_map_commands.empty());
 
-  assert(!tick_event);
```

Comment:
Arguably yes. I got rid of this since getting rid of the asserts was the suggested way to solve the problem of not having a lock on the class when reading them without making them atomic.